src/Spain/Mews.Fiscalizations.Spain/Communication/SoapClient.cs

```diff
@@ -30,7 +32,7 @@ internal SoapClient(Uri endpointUri, X509Certificate certificate, TimeSpan httpT
             where TIn : class, new()
             where TOut : class, new()
         {
-            var messageBodyXmlElement = XmlManipulator.Serialize(messageBodyObject);
+            var messageBodyXmlElement = XmlSerializer.Serialize(messageBodyObject, new XmlSerializationData(GetSiiNameSpaces()));
```

Comment:
Why do we need the namespaces now when they weren't needed before? I still don't understand why the class exists.

Reply:
https://github.com/MewsSystems/fiscalizations/blob/eebf444b2881f165afc6069d02c8cc581556f560/src/Spain/Mews.Fiscalizations.Spain/Communication/XmlManipulator.cs#L33

Reply:
Alright, I get why it's needed now :)